packages/web/app/src/pages/target.tsx

```diff
@@ -266,110 +203,86 @@ function SchemaView(props: {
     return noSchema;
   }
 
-  const canMarkAsValid = project.registryModel === RegistryModel.Legacy;
-  const showExtra = canManage;
+  const canMarkAsValid = project.registryModel === RegistryModel.Legacy && canManage;
 
-  const [open, setOpen] = useState<boolean>(false);
   const schemas = useFragment(SchemaView_SchemaFragment, target.latestSchemaVersion?.schemas.nodes);
   const compositeSchemas = schemas?.filter(isCompositeSchema) as CompositeSchema[];
-
-  const [showExtraParam, setShowExtraParam] = useState<string>('');
-  const serviceParams = router.latestLocation.search?.service;
-  if (serviceParams) {
-    setShowExtraParam(serviceParams);
-  }
-  const serviceIdFromParam = compositeSchemas?.find(
-    schema => schema.service === showExtraParam,
-  )?.id;
-  useEffect(() => {
-    if (showExtraParam && serviceIdFromParam) {
-      handleChange({ target: { value: showExtraParam } } as any);
-      setOpenItems(prevItems => [...new Set([...prevItems, serviceIdFromParam])]);
-    }
-  }, [showExtraParam]);
+  const singleSchema = schemas?.filter(schema => !isCompositeSchema(schema))[0] as
+    | SingleSchema
+    | undefined;
+  const schemasToDisplay = selectedServiceName
+    ? compositeSchemas.filter(schema => schema.service === selectedServiceName)
+    : compositeSchemas;
 
   return (
     <>
-      {showExtra ? (
```

Comment:
this condition broke the UI. If a user has no access to click "Mark as valid" button then he also cannot see the combobox.